src/modules/app/actions/listen-to-updates.js

```diff
@@ -1,101 +1,118 @@
+import BigNumber from 'bignumber.js'
 import { augur } from 'services/augurjs'
+import { updateAssets } from 'modules/auth/actions/update-assets'
+import { syncBlockchain } from 'modules/app/actions/sync-blockchain'
+import syncUniverse from 'modules/universe/actions/sync-universe'
+import { convertLogsToTransactions } from 'modules/transactions/actions/convert-logs-to-transactions'
+import { loadMarketsInfo } from 'modules/markets/actions/load-markets-info'
+import { updateOutcomePrice } from 'modules/markets/actions/update-outcome-price'
+// import { fillOrder } from 'modules/bids-asks/actions/update-market-order-book'
+import { updateMarketTopicPopularity } from 'modules/topics/actions/update-topics'
+import { updateAccountTradesData, updateAccountBidsAsksData, updateAccountCancelsData, updateAccountPositionsData } from 'modules/my-positions/actions/update-account-trades-data'
+import claimTradingProceeds from 'modules/my-positions/actions/claim-trading-proceeds'
+import * as TYPES from 'modules/transactions/constants/types'
 
 export function listenToUpdates() {
   return (dispatch, getState) => {
     augur.events.startBlockListeners({
       onAdded: (block) => {
         console.log('block added:', block)
-        // dispatch(syncBlockchain())
-        // dispatch(updateAssets())
-        // dispatch(syncUniverse())
+        dispatch(syncBlockchain())
+        dispatch(syncUniverse())
       },
       onRemoved: (block) => {
         console.log('block removed:', block)
-        // dispatch(syncBlockchain())
-        // dispatch(updateAssets())
-        // dispatch(syncUniverse())
+        dispatch(syncBlockchain())
```

Comment:
We’ll have to expand this listener as simply updating the block + universe won’t accommodate all changes of a removed block.